src/CMakeLists.txt

```diff
@@ -31,6 +31,7 @@ message(STATUS "Q4Wine build environment -- \n")
 
 set(QT_MIN_VERSION "5.2.1")
 find_package(Qt5Core REQUIRED)
+find_package(Qt5DBus REQUIRED)
```

Comment:
this one is not required unless `WITH_DBUS` is set. So the package presence is checked on line 47